app/components/captcha_submit_button_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= content_tag(
+      :'lg-captcha-submit-button',
+      **tag_options,
+      'recaptcha-site-key': IdentityConfig.store.recaptcha_site_key_v3,
+      'recaptcha-action': action,
+    ) do %>
+  <%= f.error(:recaptcha_token) %>
+  <%= f.input(:recaptcha_token, as: :hidden) %>
+  <% if show_mock_score_field? %>
```

Comment:
WDYT of moving the mock stuff to a partial? so that way all the intentionally un-18n'ed strings are in one file specifically for mocking?

Reply:
> WDYT of moving the mock stuff to a partial? so that way all the intentionally un-18n'ed strings are in one file specifically for mocking?

I kinda feel that referencing a partial from a component would break the self-contained nature of components.

Perhaps... ?

- We translate the texts
- Or, pass this from the callsite, at which point a partial would be more agreeable (though a bit more tedious to implement consistently across all phone setup screens)

Reply:
Translating makes sense

Can we put partials in the same directory as components? Or do they need to be just a single file?

Reply:
Translated texts in b319d2cff.